ibm/mas_devops/roles/suite_app_config/tasks/main.yml

```diff
@@ -101,21 +101,5 @@
 # 8. Wait for application workspace to be ready
 # -----------------------------------------------------------------------------
 - name: "Wait for application workspace to be ready ({{ mas_app_cfg_delay }}s delay)"
```

Comment:
What if we change the step name to something else than `- name: "Wait for application workspace to be ready ({{ mas_app_cfg_delay }}s delay)"` ? because the `suite_app_verify` will call the same step again, so we end up having two steps with same name?

Reply:
Agree

Reply:
Done